ibis/impala/compiler.py

```diff
@@ -334,6 +334,10 @@ def _window(translator, expr):
 def _format_window(translator, op, window):
     components = []
 
+    if window.max_lookback is not None:
+        raise NotImplementedError('Rows with max lookback is not implemented '
+                                  'for impala backends.')
```

Comment:
Let's make this `Impala-based backends`.

Reply:
Fixed in next commit